insights/parsers/podman_list.py

```diff
@@ -0,0 +1,127 @@
+"""
+PodmanList - command ``/usr/bin/podman (images|ps)``
+====================================================
+
+Parse the output of command "podman_list_images" and "podman_list_containers",
+which have very similar formats.
+
+The header line is parsed and used as the names for the remaining columns.
+All fields in both header and data are assumed to be separated by at least
+three spaces.  This allows single spaces in values and headers, so headers
+such as 'IMAGE ID' are captured as is.
+
+If the header line and at least one data line are not found, no data is
+stored.
+
+Each row is stored as a dictionary, keyed on the header fields. The data is
+available in two formats:
+
+* The old format is a list of row dictionaries.
+* The new format stores each dictionary in a dictionary keyed on the value of
+  a given field, given by the subclass.
+
+Sample output of command ``/usr/bin/podman images --all --no-trunc --digests``::
+
+    REPOSITORY                           TAG                 DIGEST              IMAGE ID                                                           CREATED             SIZE
+    rhel7_imagemagick                    latest              <none>              882ab98aae5394aebe91fe6d8a4297fa0387c3cfd421b2d892bddf218ac373b2   4 days ago          785.4 MB
+    rhel6_nss-softokn                    latest              <none>              dd87dad2c7841a19263ae2dc96d32c501ee84a92f56aed75bb67f57efe4e48b5   5 days ago          449.7 MB
+
+
+Sample output of command ``/usr/bin/podman ps --all --no-trunc --size``::
+
+    CONTAINER ID                                                       IMAGE                                                              COMMAND                                            CREATED             STATUS                        PORTS                  NAMES               SIZE
+    95516ea08b565e37e2a4bca3333af40a240c368131b77276da8dec629b7fe102   bd8638c869ea40a9269d87e9af6741574562af9ee013e03ac2745fb5f59e2478   "/bin/sh -c 'yum install -y vsftpd-2.2.2-6.el6'"   51 minutes ago      Exited (137) 50 minutes ago                          tender_rosalind     4.751 MB (virtual 200.4 MB)
+    03e2861336a76e29155836113ff6560cb70780c32f95062642993b2b3d0fc216   rhel7_httpd                                                        "/usr/sbin/httpd -DFOREGROUND"                     45 seconds ago      Up 37 seconds                 0.0.0.0:8080->80/tcp   angry_saha          796 B (virtual 669.2 MB)
+
+Examples:
```

Comment:
Better move the `Examples:` to the sub-classes separately.

Reply:
done